antlr-kotlin-runtime/src/jsMain/kotlin/com/strumenta/antlrkotlin/runtime/Environment.kt

```diff
@@ -0,0 +1,10 @@
+package com.strumenta.antlrkotlin.runtime
+
+internal actual fun platformGetEnv(name: String): String? {
+  if (isNodeJs()) {
+    return js("process.env[name]").unsafeCast<String?>()
+  }
+
+  System.out.println("Environment variables are not supported in the browser")
```

Comment:
Should this be an exception?

Reply:
No, because if I throw an exception the browser target becomes unusable.

In a browser context the `null` value signifies an environment variable is not found, and the default value (always mandatory) should be used.